omc3/model/model_creators/abstract_model_creator.py

```diff
@@ -476,33 +525,32 @@ def get_madx_script(self) -> str:
                 f"Updating the dpp is not implemented for correction model creator of {self.accel.NAME}."
             )
 
-        madx_script = self.get_base_madx_script()  # do not get_madx_script as we don't need the uncorrected output.
+        # do not get_madx_script as we don't need the uncorrected output.
+        madx_script = self.get_base_madx_script()
```

Comment:
comment placement, might also be confusing now.
```suggestion
        # use only base-part and not the full madx-script as we don't need the uncorrected output.
        madx_script = self.get_base_madx_script()
```